src/foremast/elb/create_elb.py

```diff
@@ -274,3 +277,17 @@ def configure_attributes(self, json_data):
             LOG.debug('Load Balancer Attributes:\n%s', pformat(load_balancer_attributes))
             elbclient.modify_load_balancer_attributes(
                 LoadBalancerName=self.app, LoadBalancerAttributes=load_balancer_attributes)
+
+    def add_elb_tags(self):
+        env = boto3.session.Session(profile_name=self.env, region_name=self.region)
+        elbclient = env.client('elb')
+
+        group = os.getenv("PROJECT")
+        repo = os.getenv("GIT_REPO")
+        tags = [{"Key": "Name",         "Value": '{}{}'.format(group, repo)},
+                {"Key": "app_name",     "Value": '{}{}'.format(group, repo)},
+                {"Key": "app_group",    "Value": group},
+                {"Key": "git_project",  "Value": '{}/{}'.format(group, repo)},
+                {"Key": "git_commit",   "Value": self.git_commit},
+                {"Key": "artifact_url", "Value": os.getenv("ARTIFACT_PATH")}]
```

Comment:
In general, foremast typically only tags:
app_name and app_group 

I vote to remain consistent with that. Don't like pulling environment  (using os.getenv) variables in this fashion as they are not always guaranteed.

Could we limit tagging to app_name and app_group? These are already apart of foremast as well so no need to pull them from environment variables either.

Reply:
With that being said, if you wish to extend it similar to S3 buckets implementation where users can specify additional tags, by all means! :)